redbot/cogs/warnings/warnings.py

```diff
@@ -300,20 +353,37 @@ def __init__(self, bot: Red):
         await member_settings.total_points.set(current_point_count)
 
         await warning_points_add_check(self.config, ctx, user, current_point_count)
-        try:
+        dm = await self.config.guild(ctx.guild).toggle_dm()
+        if dm:
```

Comment:
The DM feature .. are we expecting it to only send it via DM or send DM as an aditional thing?

Reply:
We expect it to send to DM as an additional thing, this is a separate setting.